File: Modules/SIE.ZYF/SIE.Web.ZYF/ProductManages/Commands/SaveProductManageCommand.js
﻿SIE.defineCommand("SIE.Web.ZYF.ProductManages.Commands.SaveProductManageCommand", {
    extend: 'SIE.cmd.FormSave',
    meta: { text: "保存", group: "edit", iconCls: "icon-SaveEntity icon-blue" },
    onSavedMsg: function (view, res) {
        var me = this;
        SIE.Msg.showInstantMessage('保存成功');
        setTimeout(function () { CRT.Workbench.closeCurrentTab(); }, 3000, me);
    }
});

File: Modules/SIE.ZYF/SIE.Web.ZYF/ProductManages/Commands/SaveProductManageCommand.cs
using SIE.Domain;
using SIE.Web.Command;
using SIE.ZYF.ProductManages;

namespace SIE.Web.ZYF.ProductManages.Commands
{
    public class SaveProductManageCommand: FormSaveCommand
    {
        protected override void DoSave(Entity entity)
        {
            if (entity is ProductManage product)
            {
                product.ModifyCount += 1;
                base.DoSave(product); // 调用父类方法
            }
        }
    }
}
